config/crd/bases/appstudio.redhat.com_integrationtestscenarios.yaml

```diff
@@ -108,8 +108,6 @@ spec:
                         required:
                         - deploymentTargetClaim
                         type: object
-                    required:
```

Comment:
Why this change?

Reply:
This is a result of the [change on the application-api side](https://github.com/redhat-appstudio/application-api/commit/1df054f63121c3860fac87679e61e2cdd1c91a15) being applied here. The `env` field is not required anymore in the latest version of the Environment configuration, and the ITS API [references the same struct](https://github.com/redhat-appstudio/integration-service/blob/3def7addb6470aec18020efc92c02a79fe3a7cc1/api/v1beta1/integrationtestscenario_types.go#L57).